tests/solver_test.py

```diff
@@ -113,26 +113,26 @@ def test_solver(self):
             '"UNDETECTED", "classifier": [["License :: OSI Approved :: Apache Software '
             'License", "Apache Software License"]], "warning": false}}, '
             '"polyaxon-client": {"0.6.0": {"license": ["MIT License", "MIT"], '
-            '"license_version": "LICENSE-WITHOUT-VERSION", "classifier": [], "warning": '
-            'false}}, "django-redis-cache": {"1.5.1": {"license": ["UNKNOWN"], '
-            '"license_version": "UNDETECTED", "classifier": [], "warning": false}}, '
-            '"django-webpack-loader": {"0.0.6.1": {"license": ["UNKNOWN"], '
-            '"license_version": "UNDETECTED", "classifier": [["License :: OSI Approved :: '
-            'MIT License", "MIT License"]], "warning": false}}, "trimesh": {"2.29.9": '
-            '{"license": ["MIT License", "MIT"], "license_version": '
-            '"LICENSE-WITHOUT-VERSION", "classifier": [["License :: OSI Approved :: MIT '
-            'License", "MIT License"]], "warning": false}}, "cwltool": '
-            '{"1.0.20160108200940": {"license": ["Apache License 2.0", "Apache-2.0", '
-            '"Apache 2.0"], "license_version": "2.0", "classifier": [], "warning": '
-            'false}}, "moto": {"1.3.15.dev221": {"license": ["Apache"], '
-            '"license_version": "UNDETECTED", "classifier": [["License :: OSI Approved :: '
-            'Apache Software License", "Apache Software License"]], "warning": false}}, '
-            '"setuptools-scm": {"1.15.4": {"license": ["MIT License", "MIT"], '
+            '"license_version": "LICENSE-WITHOUT-VERSION", "classifier": '
```

Comment:
I fetched your branch and tested pytest and solver_test is failing on `test_solver` method.

Reply:
Hmm, interesting... Did you run tests with `pytest tests/`? Because I have successful tests. Can you provide diff?

```
================================== test session starts ===================================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /home/vpodhajecky/git/thoth-station/license-solver
collected 34 items                                                                       

tests/classifier_test.py ....                                                      [ 11%]
tests/comparator_test.py ....                                                      [ 23%]
tests/json_solver_test.py ....                                                     [ 35%]
tests/license_test.py ...                                                          [ 44%]
tests/output_creator_test.py ......                                                [ 61%]
tests/package_test.py ......                                                       [ 79%]
tests/solver_test.py .......                                                       [100%]

=================================== 34 passed in 0.38s ===================================
```

Reply:
Hi @pacospace, can you look at it?